desktop/src/navigator.rs

```diff
@@ -105,6 +110,33 @@ impl NavigatorBackend for ExternalNavigatorBackend {
 
         let processed_url = self.pre_process_url(modified_url);
 
+        if processed_url.scheme() == "javascript" {
+            tracing::warn!(
+                "SWF tried to run a script on desktop, but javascript calls are not allowed"
+            );
+            return;
+        }
+
+        if self.open_url_mode == OpenURLMode::Confirm {
+            let message =
```

Comment:
Why not `format!("The SWF file wants to open the website {processed_url}")`?

Reply:
I can change it if you want.
I'd probably change it then in the WebNavigatorBackend as well for consistency.